ecom_test/tests/backend_tests/products/test_list_products_with_filter.py

```diff
@@ -0,0 +1,40 @@
+
+import pytest
+from ecom_test.src_file.api_helpers.ProductsAPIHelpers import ProductsAPIHelper
+from ecom_test.src_file.dao.products_dao import ProductsDAO
+from datetime import datetime, timedelta
+
+
+@pytest.mark.regression
+class TestListProductsWithFilter(object):
+
+    # create a method to get the list of products with filter for specific period of time from now
+
+    @pytest.mark.tcid35
+    def test_list_products_with_filter_after(self):
+        # Create a payload data request to be sent
+        x_days_from_now = 60  # variable for requesting data for last 30 days form now
```

Comment:
You are doing 60 but the comment says 30

Reply:
changed